sqlmesh/core/engine_adapter/snowflake.py

```diff
@@ -91,6 +87,12 @@ def session(self, properties: SessionProperties) -> t.Iterator[None]:
         yield
         self.execute(f"USE WAREHOUSE {current_warehouse_sql}")
 
+    @property
+    def current_warehouse(self) -> exp.Identifier:
```

Comment:
[Nit] Looks like an implementation detail to me. Should this be `_current_warehouse`?

Reply:
Ah yeah I just copied the same approach as the `snowpark` property. Have updated